git_machete/client.py

```diff
@@ -2387,6 +2383,91 @@ def __sync_before_creating_pr(self, *, opt_onto: Optional[LocalBranchShortName],
                 return
             raise MacheteException('Pull request creation interrupted.')
 
+    def __add_branch(self,
+                     *,
+                     branch: LocalBranchShortName,
+                     opt_onto: Optional[LocalBranchShortName],
+                     opt_as_root: bool,
+                     opt_yes: bool,
```

Comment:
Hmmm AFAICS this is always `True` right? better to refactor it away then

Reply:
Or rather, generally:
1. if this method is likely to be reused across `git machete github checkout-prs` and `git machete add`, let's keep it generic BUT also use it from `def add`
2. alternatively, if this method will only serve the purposes of `git machete github checkout-prs`, let's keep it as simple and UNconfigurable as possible

Reply:
Alright, i have chosen to go for option 1.